client/server/pages/index.js

```diff
@@ -485,6 +485,13 @@ function setUpCSP( req, res, next ) {
 }
 
 function setUpRoute( req, res, next ) {
+	if ( req.context.isRouteSetup === true ) {
+		console.log( '\nRoute already setup. Ambiguous route definition detected.' );
```

Comment:
Open to more ideas about alerting here. Should we log2logstash here for better visibility?

Reply:
I think using `log2logstash()` is a good idea here to track how many times this occurred.